Source/Core/Core/HW/Sram.h

```diff
@@ -34,55 +34,102 @@ distribution.
 -------------------------------------------------------------*/
 #pragma once
 
+#include <array>
 #include "Common/CommonTypes.h"
 
+typedef std::array<u8, 12> CardFlashId;
```

Comment:
```using```?

Reply:
i don't see the point, but i've changed it to `using` anyways...